app/views/passengers/show.html.erb

```diff
@@ -0,0 +1,65 @@
+<h2><%= @passenger.name%></h2>
+
+<section class="">
+  <% if @passenger.ride_in_progress %>
+    <h3>Ride in Progress... </h3>
+
+  <% elsif Driver.first_available_driver == nil %>
+    <h3>No Driver Is Available At This Time</h3>
+
+  <% else %>
+    <%= link_to "Request a Ride", passenger_trips_path(@passenger.id), method: :post %>
+  <% end %>
+</section>
+
+<p>Phone Number: <%= @passenger.phone_num %></p>
+<p>Total Charged: $<%= @passenger.total_charge %></p>
+<h3>Trips: </h3>
+
+<% @trips.each do |trip| %>
+<section class="trips-table">
```

Comment:
The styling on this page is a little weird when the size of the screen gets smaller.  The highlighted portion shrinks, but the text is still there if you scroll to the right.